x-pack/plugins/ml/public/application/components/job_spaces_repair/repair_results.tsx

```diff
@@ -0,0 +1,178 @@
+          </EuiTextColor>
+        </h3>
+      </EuiTitle>
+      <EuiText size="s">
+        <p>
+          <EuiTextColor color="subdued">
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsDeleted.description"
+              defaultMessage="Saved objects which do not have an accompanying job. These will be left over when deleting anomaly detection jobs."
+            />
+          </EuiTextColor>
+        </p>
+      </EuiText>
+    </>
+  );
+  return <RepairItem id="savedObjectsDeleted" title={title} items={items} />;
+};
+
+const DatafeedsAdded: FC<{ repairItems: RepairSavedObjectResponse }> = ({ repairItems }) => {
+  const items = Object.keys(repairItems.datafeedsAdded);
+
+  const title = (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          <EuiTextColor color={items.length ? 'default' : 'subdued'}>
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.datafeedsAdded.title"
+              defaultMessage="Job saved objects with missing datafeeds ({count})"
+              values={{ count: items.length }}
+            />
+          </EuiTextColor>
+        </h3>
+      </EuiTitle>
+      <EuiText size="s">
+        <p>
+          <EuiTextColor color="subdued">
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.datafeedsAdded.description"
+              defaultMessage="Anomaly detection job saved objects which are missing their datafeed id."
+            />
+          </EuiTextColor>
+        </p>
+      </EuiText>
+    </>
+  );
+  return <RepairItem id="datafeedsAdded" title={title} items={items} />;
+};
+
+const DatafeedsRemoved: FC<{ repairItems: RepairSavedObjectResponse }> = ({ repairItems }) => {
+  const items = Object.keys(repairItems.datafeedsRemoved);
+
+  const title = (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          <EuiTextColor color={items.length ? 'default' : 'subdued'}>
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.datafeedsRemoved.title"
+              defaultMessage="Job saved objects with unmatched datafeed ids ({count})"
+              values={{ count: items.length }}
+            />
+          </EuiTextColor>
+        </h3>
+      </EuiTitle>
+      <EuiText size="s">
+        <p>
+          <EuiTextColor color="subdued">
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.datafeedsRemoved.description"
+              defaultMessage="Anomaly detection jobs which contain a datafeed ID, but the datafeed does not exist."
+            />
+          </EuiTextColor>
+        </p>
+      </EuiText>
+    </>
+  );
+  return <RepairItem id="datafeedsRemoved" title={title} items={items} />;
+};
+
+const RepairItem: FC<{ id: string; title: JSX.Element; items: string[] }> = ({
+  id,
+  title,
+  items,
+}) => (
+  <EuiAccordion id={id} buttonContent={title} paddingSize="l">
```

Comment:
@elastic/kibana-design Do we need to ask EUI for a disabled state for accordions for when there are no items here? Just changing the color of the text doesn't seem to be the best option when it leaves a button that doesn't do anything...

Reply:
It's worth opening an EUI 'Discuss' issue, aside from this PR.
For the sake of this PR, and lacking the existing of a disabled state, I would propose adding some copy with the accordion content body state something to the affect of 'No results found'.

Reply:
EUI Discussion for a generic solution: https://github.com/elastic/eui/discussions/4276